crates/provider/Cargo.toml

```diff
@@ -45,6 +45,13 @@ tokio = { workspace = true, features = ["macros"] }
 tracing-subscriber = { workspace = true, features = ["fmt"] }
 
 [features]
+default = ["http"]
 pubsub = ["alloy-rpc-client/pubsub", "dep:alloy-pubsub"]
+http = [
```

Comment:
how does this interact with hyper?

Reply:
Interesting question! Right now it doesn't iteract with hyper at all... But I noticed features "[reqwest](https://github.com/alloy-rs/alloy/blob/7e39c85f9f51e6449a8b661f54df0ac213f18639/crates/provider/src/builder.rs#L228-L229)" and "[hyper](https://github.com/alloy-rs/alloy/blob/7e39c85f9f51e6449a8b661f54df0ac213f18639/crates/provider/src/builder.rs#L238-L239)" has been added in #342 but they aren't declared in the Cargo.toml.

Should I add these 2 new features that both enable "http" ?

Reply:
Yes, that would be my preferred approach 👍